### DISCUSSION TITLE
Missing some functions when instrument a Spring project

### DISCUSSION THREAD

**Discussion by amirdeljouyi** [btraceio/btrace#624] [Q&A]
Hello,

I'm instrumenting a Spring project, and I noticed that some methods like Controller and Services don't show up in the trace log. They call a repository method in which Pageable class is used and when I remove Pageable everything works fine. 

```
ArrayList<Owner> findAll(Pageable pageable); // when remove Pageable, the parent method will be traced
```

So far, I've attached the trace script dynamically to the running application. As well, I wanted to try the trace script with the application when it is started to see if it would help. So I've tried -javaagent and Btracer, but then they don't show any logs. Also no dump classes are generated.

Best,
Amir

**Comment by jbachorik** [comment]
Would it be possible to create a simple reproducer app you could attach here so i can try and diagnose?
